### DISCUSSION TITLE
Using Pro and Free version at once

### DISCUSSION THREAD

**Discussion by Riki007B** [six7/figma-tokens#1098] [Q&A] [answered]
Hi, I am wondering - is it possible to use both Free and Pro version for different roles of users?
We would like to use new features of PRO version for designers who are creating components and prototypes.
But we would like to also use Free version for developers, who are implementing content and they just need to check where and which token is used - they do not need such features. Will these versions be compatible?
(We are using JSONbin as repository for tokens)

And my second question - is there any way to lock editing of tokens for some users/roles?
Many thanks in advance.

**Comment by rbosker** [comment, marked as answer]
Hi @Riki007B , it is possible to use both free and pro next to each other. The free users only only have read access to the Pro features. In your case this should work.

It is also possible to share the tokens as read-only. Here you can find more info on the read-only for JSONbin: https://docs.figmatokens.com/sync/jsonbin#read-only-mode

I hope this answers your question!

**Reply by Riki007B** [reply]
Thank you!
